buildSrc/src/main/kotlin/splunk.java-conventions.gradle.kts

```
@@ -13,11 +13,11 @@ repositories {
   }
 }
 
-val otelVersion = "1.19.0"
-val otelAlphaVersion = "1.19.0-alpha"
-val otelContribAlphaVersion = "1.19.1-alpha"
-val otelInstrumentationVersion = "1.19.1"
-val otelInstrumentationAlphaVersion = "1.19.1-alpha"
+val otelVersion = "1.20.0-SNAPSHOT"
+val otelAlphaVersion = "1.20.0-alpha-SNAPSHOT"
```

These two are SDK versions, right? Shouldn't they stay at 1.19?
```suggestion
val otelVersion = "1.19.0"
val otelAlphaVersion = "1.19.0-alpha"
```